### PR TITLE
[ko] Update package-objects.md

### PR DESCRIPTION
**Pull request by chipmunklimo**
updated Package Objects page translated in Korean

### CODE REVIEW

**Comment by eed3si9n** [comment]
@chipmunklimo Thanks for the contribution!

@scala/docs-ko Review please?

File: _ko/tour/package-objects.md [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+object PrintPlanted {
+  def main(args: Array[String]): Unit = {
+    for (fruit <- planted) {
+      showFruit(fruit)
+    }
+  }
+}
+```
+
+패키지 객체는 이를 만들기 위해 상속을 이용할 수 있다는 점에서 다른 객체와 같다. 그 예로, 한 패키지 객체에 여러 트레잇을 더할 수 있다:
```
Comment (eugene70) [inline _ko/tour/package-objects.md:61]: 패키지 객체를 만들기 위해 다른 객처처럼 상속을 이용할 수 있습니다. 예를 들면, 다음처럼 여러 트래잇을 혼합하여 구성할 수 있습니다:

**Comment by eugene70** [comment]
Before reviewing, I am not a Scala expert & translator.

I think we would use honorifics like '제공합니다' instead '제공한다'.

**Comment by eed3si9n** [comment]
The level of honorifics should be consistent with the rest of the tour. Looking at https://docs.scala-lang.org/ko/tour/tour-of-scala.html, the sentences end in 니다 so that's the polite style?

File: _ko/tour/package-objects.md [hunk 2]
```diff
@@ -1,14 +1,67 @@
...
+package gardening
+package object fruits {
+  val planted = List(Apple, Plum, Banana)
+  def showFruit(fruit: Fruit): Unit = {
+    println(s"${fruit.name}s are ${fruit.color}")
+  }
+}
+```
+
+아래 사용 방법의 예시처럼, `PrintPlanted` 객체는 `gardening.fruits` 패키지에 와일드카드를 사용하여 임포트함으로써 `Fruit` 클래스를 불러왔으며, 이와 정확히 같은 방식으로 `planted`와 `showFruits`를 불러온다:
```
Comment (eugene70) [inline _ko/tour/package-objects.md:47]: 아래 사용예처럼, `PrintPlanted` 객체는 `gardening.fruits` 패키지에 와일드카드를 사용하여 임포트 함으로써 `Fruit` 클래스와 함께 `planted`와 `showFruits`도 불러옵니다:

File: _ko/tour/package-objects.md [hunk 3]
```diff
@@ -1,14 +1,67 @@
...
-# Package objects
+# 패키지 객체
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
+스칼라는 패키지 객체라는 패키지 전체에 걸쳐 공유되는 편리한 컨테이너를 제공한다.
+
+패키지 객체는 변수나 메서드 정의뿐 아니라 임의의 정의도 담을 수 있다. 예를 들어, 패키지 범위의 타입 별칭이나 암시적 변환을 담기 위해 패키지 객체가 종종 사용된다. 패키지 객체는 스칼라 클래스나 트레잇을 상속할 수도 있다.
+
+관례로 패키지 객체의 소스 코드는 `package.scala`라 명명된 소스 파일에 담긴다.
```
Comment (eugene70) [inline _ko/tour/package-objects.md:17]: 패키지 객체의 소스 코드는 `package.scala`라 명명된 소스 파일에 담는 것이 관례입니다.

File: _ko/tour/package-objects.md [hunk 4]
```diff
@@ -1,14 +1,67 @@
...
+
+각 패키지는 하나의 패키지 객체를 가질 수 있다. 패키지 객체에 정의된 것은 무엇이든 해당 패키지의 구성원으로 간주한다.
+
+아래 예제를 보자. 먼저 `gardening.fruits` 패키지에 하나의 `Fruit` 클래스와 세 개의 `Fruit` 객체가 있다고 가정하자:
```
Comment (eugene70) [inline _ko/tour/package-objects.md:21]: 아래의 예시를 보십시오. 먼저 `gardening.fruits` 패키지에 `Fruit` 클래스와 세 개의 `Fruit` 객체가 있다고 가정해보겠습니다:

**Comment by eugene70** [comment]
We can refer to this similar translating project: https://github.com/javascript-tutorial/ko.javascript.info

**Comment by chipmunklimo** [comment]
> > The level of honorifics should be consistent with the rest of the tour. Looking at https://docs.scala-lang.org/ko/tour/tour-of-scala.html, the sentences end in 니다 so that's the polite style?
> 
> Yes, I think. But the documents are a mixture of the two styles. We should change that.

The style of this document might seem not too polite but not that rude as well. Most Korean tech books are following this and personally it is better to read because of simplicity.
I would like to suggest to change the style after a consensus of the community. For now, I will keep this style for consistency.

**Comment by eugene70** [comment]
> > > The level of honorifics should be consistent with the rest of the tour. Looking at https://docs.scala-lang.org/ko/tour/tour-of-scala.html, the sentences end in 니다 so that's the polite style?
> > 
> > 
> > Yes, I think. But the documents are a mixture of the two styles. We should change that.
> 
> The style of this document might seem not polite but not that rude as well. Most Korean tech books are following this and personally it is better to read because of its simplicity.
> I would like to suggest to change the style after a consensus of the community. For now, I will keep this style for consistency.

It is worth discussing. On the contrary, I think most technical books use polite expressions.
What's more, I think this should be even more so with a tutorial-like guide for the public. The existing expression is like a paper and feels like a dry monologue. You can also refer to other tutorials as like:

  - https://github.com/javascript-tutorial/ko.javascript.info
  - https://wiki.ubuntu-kr.org/index.php/Getting_Started
  - https://github.com/KerasKorea/KEKOxTutorial

Unfortunately, there seems to be no community yet to discuss this.

File: _ko/tour/package-objects.md (hunk 1 above)
Comment (chipmunklimo) [reply _ko/tour/package-objects.md:61]: 패키지 객체를 만들기 위해 다른 객체처럼 상속을 이용할 수 있다. 다음 예시처럼 여러 트레잇을 혼합하여 구성할 수 있다:

File: _ko/tour/package-objects.md (hunk 2 above)
Comment (chipmunklimo) [reply _ko/tour/package-objects.md:47]: 아래 사용 예처럼, `PrintPlanted` 객체는 `gardening.fruits` 패키지에 와일드카드를 사용하여 임포트 함으로써 `Fruit` 클래스와 함께 `planted`와 `showFruits`도 불러온다:

File: _ko/tour/package-objects.md (hunk 3 above)
Comment (chipmunklimo) [reply _ko/tour/package-objects.md:17]: 패키지 객체의 소스 코드는 `package.scala`라 명명된 소스 파일에 담는 것이 관례이다.

File: _ko/tour/package-objects.md (hunk 4 above)
Comment (chipmunklimo) [reply _ko/tour/package-objects.md:21]: 아래 예제를 보자. 먼저 `gardening.fruits` 패키지에 `Fruit` 클래스와 세 개의 `Fruit` 객체가 있다고 가정한다:

**Comment by chipmunklimo** [comment]
> > > > The level of honorifics should be consistent with the rest of the tour. Looking at https://docs.scala-lang.org/ko/tour/tour-of-scala.html, the sentences end in 니다 so that's the polite style?
> > > 
> > > 
> > > Yes, I think. But the documents are a mixture of the two styles. We should change that.
> > 
> > 
> > The style of this document might seem not polite but not that rude as well. Most Korean tech books are following this and personally it is better to read because of its simplicity.
> > I would like to suggest to change the style after a consensus of the community. For now, I will keep this style for consistency.
> 
> It is worth discussing. On the contrary, I think most technical books use polite expressions.
> What's more, I think this should be even more so with a tutorial-like guide for the public. The existing expression is like a paper and feels like a dry monologue. You can also refer to other tutorials as like:
> 
> * https://github.com/javascript-tutorial/ko.javascript.info
> * https://wiki.ubuntu-kr.org/index.php/Getting_Started
> * https://github.com/KerasKorea/KEKOxTutorial
> 
> Unfortunately, there seems to be no community yet to discuss this.

I see, respect your opinion but we still have a consistency issue. You might be able to register it as an issue besides this one.
please read few comments above I rewrote. Thank you for reviewing.
덕분에 문장이 훨씬 매끄러워졌습니다. 리뷰 감사드립니다^^

**Pull request merged**